src/collections/_documentation/performance/discover/query-builder.md

```diff
@@ -314,7 +314,24 @@ A down arrow indicates sorting the column in descending order. An up arrow indic
 
 ### Download CSV
 
-Note, only the current values visible on the table are exported, not the paginated results. To export the table as CSV, click on the "Download CSV" button.
+If you want to take the data elsewhere, click on the "Export" button to start the export. The results are going to take a while. You'll get an email with the download link once they're ready.
```

Comment:
Can we say "depending on the file size, the results can take between one second to one hour"

Reply:
Would that be misleading if the results can take well over an hour? I've seen it take like 90 minutes, and there isn't really a strict upper bound as how long it'll take so it can potentially go above that.

Reply:
How about:

Depending on the file size, the result times will vary.

Reply:
Is that statement accurate? I'm assuming the results are dependent on the file size.

Reply:
Can we can say

Depending on the amount of data, the wait times will vary.

Reply:
👍 That's great! Thanks, Tony

Reply:
```suggestion
If you want to take the data elsewhere, click on the "Export" button to start the export. Depending on the amount of data, the wait times will vary. You'll get an email with the download link once they're ready.
```